libs/barista-components/table/src/table.ts

```diff
@@ -62,6 +62,7 @@ import {
   _COALESCED_STYLE_SCHEDULER,
 } from '@angular/cdk/table';
 import { ViewportRuler } from '@angular/cdk/scrolling';
+import { DtTableDataSource } from '..';
```

Comment:
Please do not import from a barrel file here. Will fix that for you during cleanup, but this would lead to cyclic dependencies as this files is also exported in the barrel.